via/static/scripts/video_player/components/YouTubeVideoPlayer.tsx

```diff
@@ -79,6 +95,9 @@ export default function YouTubeVideoPlayer({
   const onPlayingChangedCallback = useRef<(playing: boolean) => void>();
   onPlayingChangedCallback.current = onPlayingChanged;
 
+  const onTimeChangedCallback = useRef<(timestamp: number) => void>();
+  onTimeChangedCallback.current = onTimeChanged;
```

Comment:
What's the point of "wrapping" the callback in a ref? 🤔

Reply:
This is to avoid re-running the effect unnecessarily if the callback changes. Instead we just want to run the effect once and then use whatever the last value was.

Reply:
I added a comment to explain this for future readers.